assets/javascripts/omise-payment-form-handler.js

```diff
@@ -14,15 +14,15 @@
 		let $ulError = $("<ul>").addClass("woocommerce-error");
 
 		if ($.isArray(message)) {
-			$.each(message, function(i, v) {
+			$.each(message, function (i, v) {
```

Comment:
@aashishgurung where is our JavaScript code style defined? Can we set a `package.json` in this and all projects that have JavaScript to ensure a consistent style?

Reply:
And then do a one-shot reformatting?

Reply:
But we can can keep the formatting changes in this PR.

Reply:
But we can can keep the formatting changes in this PR.

Reply:
I have added package.json with Eslint as linter. For now, I just formatted the file changed in this PR.